patches/server/0103-Send-nearby-packets-from-world-player-list-not-serve.patch

```diff
@@ -0,0 +1,122 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: mechoriet <kevinworm92@gmail.com>
+Date: Sat, 29 Apr 2023 20:18:25 +0200
+Subject: [PATCH] =?UTF-8?q?Send=20nearby=20packets=20from=20world=20player?=
```

Comment:
wtf?
###
There are also several unnecessary, missing, or incorrect comments in this patch

Reply:
solved name no clue why it did that also made the name a bit more clear